tests/check_amb_kf.c

```diff
@@ -67,12 +68,73 @@ START_TEST(test_lsq)
 }
 END_TEST
 
+START_TEST(test_sos_innov)
```

Comment:
In terms of test coverage, I think it would be useful to have a filter test case using actual test data. Under ideal circumstances, we'd have a case with the KF from SITL where incorporating an outlier observation caused the original filter to fail.